src/lib/viewers/controls/annotations/AnnotationsControls.scss

```diff
@@ -9,3 +9,8 @@
 .bp-AnnotationsControls-exitBtn {
     display: none;
 }
+
+.ba-Layer--popup,
+.ba-Layer--highlight {
+    z-index: 3;
```

Comment:
Why do we need to set it to 3? And is there any other elements that need to be changed now that this z-index has changed?

Reply:
We need it to be above the text layer, which is now at z-index 2 in this new version of PDFjs (see https://cdnjs.cloudflare.com/ajax/libs/pdf.js/3.6.172/pdf_viewer.css - the z-index in .textLayer)

Reply:
I don't think so; this will move the search controls, the tooltip, and the modal that pops up when clicking on the tooltip

Reply:
I had to do something similar when I first merged the changes for this pdf.js version.

Reply:
I think the drawing layer is a separate layer so that may also need to move up, but should we also capture that in this ticket? Or should we have a separate ticket for that?